account_move_line_cumulated_balance/models/account_move_line.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 Tecnativa - César A. Sánchez
+# Copyright 2022 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from psycopg2 import sql
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    cumulated_balance = fields.Monetary(
+        string="Cumulated Balance",
+        store=False,
+        currency_field="company_currency_id",
+        compute="_compute_cumulated_balance",
+        help="Cumulated balance depending on the domain and the order chosen in the view.",
+    )
+    cumulated_balance_currency = fields.Monetary(
+        string="Cumulated Balance in Currency",
+        store=False,
+        currency_field="currency_id",
+        compute="_compute_cumulated_balance",
+        help="Cumulated balance in currency depending on the domain and "
+        "the order chosen in the view.",
+    )
+
+    @api.model
+    def search_read(self, domain=None, fields=None, offset=0, limit=None, order=None):
+        # Make an explicit order because we will need to reverse it
+        order = (order or self._order) + ", id"
```

Comment:
Shouldn the order key go as something like `", id DESC"` or `", id ASC"`. What if `id` is already in the order string?